etsin_finder/frontend/js/components/qvain/fields/temporalAndSpatial/temporal/temporalFieldContent.jsx

```diff
@@ -19,38 +19,30 @@ const TemporalFieldContent = ({ Store, lang }) => {
 
   return (
     <>
-      <Observer>
-        {() => (
-          <TemporalList
-            lang={lang}
-            temporals={Field.storage}
-            remove={uiid => Field.removeTemporal(uiid)}
-            readonly={Field.readonly}
-          />
-        )}
-      </Observer>
+      <TemporalList
+        lang={lang}
+        temporals={Field.storage}
+        remove={uiid => Field.removeTemporal(uiid)}
+        readonly={Field.readonly}
+      />
       <DurationPicker
         Store={Store}
         Field={Field}
         translationsRoot={translationsRoot}
         datum="duration"
         id="temporal-period"
       />
-      <Observer>
-        {() => (
-          <>
-            <Translate component={ValidationError} content={Field.validationError} />
-            <ButtonContainer>
-              <Translate
-                component={AddNewButton}
-                content={`${translationsRoot}.addButton`}
-                onClick={handleClick}
-                disabled={Field.readonly}
-              />
-            </ButtonContainer>
-          </>
-        )}
-      </Observer>
+      <>
+        {Field.validationError && <ValidationError>{Field.validationError}</ValidationError>}
+        <ButtonContainer>
+          <Translate
+            component={AddNewButton}
+            content={`${translationsRoot}.addButton`}
+            onClick={handleClick}
+            disabled={Field.readonly}
+          />
+        </ButtonContainer>
+      </>
```

Comment:
The `<>` is redundant here.
```suggestion
        {Field.validationError && <ValidationError>{Field.validationError}</ValidationError>}
      <ButtonContainer>
        <Translate
          component={AddNewButton}
          content={`${translationsRoot}.addButton`}
          onClick={handleClick}
          disabled={Field.readonly}
        />
      </ButtonContainer>
```